wls-gui-wahllokalsystem/src/components/ergebnisermittlung/OBW/stapelA/TheOBWStapelACard.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <v-card>
+    <v-card-title>Gültige Stimmabgaben</v-card-title>
+    <v-form v-model="isFormValid">
+      <v-card-text>
+        <v-table>
+          <thead>
+            <tr>
+              <th />
+              <th class="font-weight-bold">Wahlvorschlag</th>
+              <th class="font-weight-bold">Stapel a: zweifelsfrei gültig</th>
+            </tr>
+          </thead>
+          <tbody>
+            <base-row-obw-stapel-a
+              v-for="(
+                ergebnisWithWahlvorschlag, index
+              ) in ergebnisseWithWahlvorschlag"
+              :key="index"
+              :model-value="ergebnisWithWahlvorschlag.ergebnis"
+              :wahlvorschlag="ergebnisWithWahlvorschlag.wahlvorschlag"
+            />
+          </tbody>
+          <tfoot>
+            <tr>
+              <td />
+              <td class="font-weight-bold">Gültige Stimmen insgesamt</td>
+              <td class="font-weight-bold">{{ sumOfValidVotes }}</td>
+            </tr>
+          </tfoot>
+        </v-table>
+      </v-card-text>
+      <v-card-actions>
+        <base-button-save
+          :disabled="!isFormValid"
+          @click="onSaveClicked"
+        />
+      </v-card-actions>
+    </v-form>
+  </v-card>
+</template>
+
+<script setup lang="ts">
+import type { Ergebnis } from "@/types/ergebnismeldung/Ergebnis.ts";
+import type { Wahlvorschlag } from "@/types/wahlvorschlaege/Wahlvorschlag.ts";
+
+import { computed, ref } from "vue";
+
+import BaseButtonSave from "@/components/common/buttons/BaseButtonSave.vue";
+import BaseRowObwStapelA from "@/components/ergebnisermittlung/OBW/stapelA/BaseRowOBWStapelA.vue";
+import { useLogging } from "@/composables/common/logging.ts";
+import { useErgebnisUtils } from "@/composables/ergebnismeldung/ergebnisUtils.ts";
+import { useErgebnismeldungStore } from "@/stores/ergebnismeldungStore.ts";
+import { useWahlvorschlaegeStore } from "@/stores/wahlvorschlaegeStore.ts";
+import { StapelArtEnum } from "@/types/ergebnismeldung/StapelArtEnum.ts";
+
+const ergebnismeldungsStore = useErgebnismeldungStore();
+const wahlvorschlaegeStore = useWahlvorschlaegeStore();
```

Comment:
bitte direktes destructuring nutzen

```suggestion
const { sendErgebnisseByStapelArt, getErgebnisseByWahlIdAndStapelartOrUndefined } = useErgebnismeldungStore();
const { getWahlvorschlagOrUndefinedByWahlIDWahlbezirkIDAndWahlvorschlagID } = useWahlvorschlaegeStore();
```

Reply:
mit 58a443ba8e15351461917b92353e00a55be31858 die Aufrufe angepasst